src/main/java/seedu/intrack/logic/commands/SortCommand.java

```diff
@@ -23,6 +26,8 @@ public class SortCommand extends Command {
 
     public static final String MESSAGE_SUCCESS_D = "Sorted all internships in descending order";
 
+    public static final String MISSING_TASKLIST = "One or more of the internships have no tasks! Cannot be sorted.";
```

Comment:
In a recent PR #125, it is now possible to have the ```Internship``` with an empty taskList. Perhaps the code could be updated, perhaps there would be sorting amongst ```Internships```s with ```Task```s, but the ones that do not have ```Task``` will be put at the bottom in an arbitrary order.

Reply:
I am not sure if I can finish it before the feature freeze but I will try.